org.civicrm.reqangular/src/common/directives/horizontal-scroll-shadows.directive.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-env amd */
+/* globals MutationObserver */
+
+define([
+  'common/angular',
+  'common/modules/directives'
+], function (angular, directives) {
+  'use strict';
+
+  directives.directive('horizontalScrollShadows', [function () {
```

Comment:
i think `x-scroll-shadows` would be less of a mouthful

Reply:
`x-scroll-shadows` won't work

![image](https://user-images.githubusercontent.com/3973243/36897793-e99ed756-1e0f-11e8-94d4-3bd028632201.png)

Reply:
Now it is `h-scroll-shadows`

Reply:
why not just `scroll-shadows` ?